docs/setup/additional/rackawareness.md

```diff
@@ -1,16 +1,17 @@
-## Rackawareness
+# Rackawareness
 
-### How to enable
+## How to enable
 
 1. Build rack utils docker image as below
-    ```
+
+    ```sh
     docker build utilities/rackuitls/ --network host --build-arg AppName="RackUtils" -t "hbase-rack-utils:1.0.0"
     docker push hbase-rack-utils:1.0.0
     ```
 
-1. Enable sidecar along with hmaster container as described below in values.yaml file
+2. Enable sidecar along with hmaster container as described below in values.yaml file
```

Comment:
If you retain it to be 1. it will auto increment the integer.

Reply:
Ohh yeah, reverted back these changes.